cmd/kaniko-docker/main.go

```diff
@@ -116,6 +122,7 @@ func run(c *cli.Context) error {
 			Target:     c.String("target"),
 			Repo:       c.String("repo"),
 			Labels:     c.StringSlice("custom-labels"),
+			SkipTlsVerify: c.String("skip_tls_verify"),
```

Comment:
once you chance to `cli.BoolT` you can use `c.Bool("skip-tls-verify")` which returns a bool value

Reply:
done